package.json

```diff
@@ -52,9 +52,9 @@
     "vscode-test-utils": "^0.0.8"
   },
   "scripts": {
-    "precompile": "node ./node_modules/vscode/bin/install",
-    "compile": "tsc",
-    "lint": "tslint --project tsconfig.json",
+    "postinstall": "node ./node_modules/vscode/bin/install",
+    "compile": "tsc -p ./",
+    "lint": "tslint 'src/**/*.ts' 'test/**/*.ts'",
```

Comment:
What was wrong with `--project tsconfig.json`? The way I had it, you don't have to duplicate the include paths.